auslib/web/admin/swagger/api.yaml

```diff
@@ -1832,11 +1832,12 @@ paths:
                   headerArchitecture: null
```

Comment:
Yeah, I think that's the best way to do it. We should probably change the spec/API versions to "1.0" for now, instead of inheriting the Balrog app version.